doc/contributing.rst

```diff
@@ -253,42 +253,98 @@ How to make a release
 
 To make a new release of the package, follow these steps:
 
-1. Check that the nightly build on CircleCI was successful
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+1. Check the tests on GitHub Actions and CircleCI
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Check the ``nightly`` `build on
-CircleCI <https://circleci.com/gh/ESMValGroup/ESMValCore/tree/master>`__.
-All tests should pass before making a release.
+Check the ``nightly``
+`build on CircleCI <https://circleci.com/gh/ESMValGroup/ESMValCore/tree/master>`__
+and the
+`GitHub Actions run <https://github.com/ESMValGroup/ESMValCore/actions>`__.
+All tests should pass before making a release (branch).
 
-2. Make a pull request to increase the version number
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+2. Increase the version number
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The version number is stored in ``esmvalcore/_version.py``,
-``package/meta.yaml``, ``CITATION.cff``. Make sure to update all files. See
-https://semver.org for more information on choosing a version number.
-
-3. Make the release on GitHub
+``package/meta.yaml``, ``CITATION.cff``. Make sure to update all files.
+Also update the release date in ``CITATION.cff``.
+See https://semver.org for more information on choosing a version number.
+Make a pull request and get it merged into ``master``.
+
+3. Add release notes
+~~~~~~~~~~~~~~~~~~~~
+Use the script
+`esmvaltool/utils/draft_release_notes.py <https://docs.esmvaltool.org/en/latest/utils.html#draft-release-notes-py>`__
+to create create a draft of the release notes.
+This script uses the titles and labels of merged pull requests since the
+previous release.
+Review the results, and if anything needs changing, change it on GitHub and
+re-run the script until the changelog looks acceptable.
+Copy the result to the file ``doc/changelog.rst``.
+Make a pull request and get it merged into ``master``.
```

Comment:
worth saying they should add at least two of the Core Team as PR reviewers

Reply:
I should hope that someone making the release knows how to get something merged. However, maybe we should always have the release manager of the previous release as a 'backup release manager', doing it toghether like we are for this release. That would ensure some continuity.

Reply:
OK, I added this